deploy/crds/hostpathprovisioner_cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: hostpathprovisioner.kubevirt.io/v1alpha1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+spec:
+  imagePullPolicy: IfNotPresent
+  imageRegistry: quay.io/kubevirt
```

Comment:
We decided to host on quay?

Reply:
I talked to Fabian, and we are doing all new things on quay.